changelogs/fragments/support_version_parameter.yaml

```
@@ -0,0 +1,4 @@
+---
+breaking_changes:
+  - This PR adds support for version parameter for httpapi and network_cli connections. Going forward the default version
+    would be "latest" for both httpapi and network_cli.
```

This is going to end up in the collection changelog, so "This PR" will not make a lot of sense after the PR is merged. We do have a guide for writing fragments (https://docs.ansible.com/ansible/devel/community/development_process.html#changelog-fragment-entry-format) and I have tried to make it clearer for someone unaware of this change what to expect, but I am not great at writing changelog fragments and this is a rather hard change to describe succinctly.

```suggestion
  - eos_command - new suboption ``version`` of parameter ``command``, which controls the JSON response version.
    Previously the value was assumed to be "latest" for network_cli and "1" for httpapi, but the default will now be "latest" for both connections.
    This option is also available for use in modules making their own device requests with
    `plugins.module_utils.network.eos.eos.run_commands()` with the same new default behavior.
    (https://github.com/ansible-collections/arista.eos/pull/258).
```